corehq/apps/reports/filters/forms.py

```diff
@@ -9,7 +9,7 @@
 
 # For translations
 from django.utils.translation import ugettext as _
-from django.utils.translation import ugettext_noop
+from django.utils.translation import ugettext_noop, ugettext_lazy
```

Comment:
Yeah. I played it safe and didn't change all of them. Just the ones that were reported and I could spot check. Thought it may take quite a while to hunt down all of these translations that use noop